src/Plugin/EntityBrowser/SelectionDisplay/NewDisplay.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * Contains \Drupal\entity_browser\Plugin\EntityBrowser\SelectionDisplay\NewDisplay.
+ */
+
+namespace Drupal\entity_browser\Plugin\EntityBrowser\SelectionDisplay;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\entity_browser\SelectionDisplayBase;
+
+/**
+ * Show current selection and delivers selected entities.
+ *
+ * @EntityBrowserSelectionDisplay(
+ *   id = "new_display",
+ *   label = @Translation("New selection display"),
+ *   description = @Translation("Show current selection display and delivers selected entities.")
+ * )
+ */
+class NewDisplay extends SelectionDisplayBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return array(
+      'view' => NULL,
+      'view_display' => NULL,
+    ) + parent::defaultConfiguration();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getForm(array &$original_form, FormStateInterface $form_state) {
+    $selected_entities = $form_state->get(['entity_browser', 'selected_entities']);
+
+    $form = [];
+    $form['#attached']['library'][] = 'entity_browser/new_display';
+    $form['selected'] = [
+      '#theme_wrappers' => ['container'],
+      '#attributes' => ['class' => ['selected-entities-list']],
+    ];
+    $form['selected']['weights'] = [
```

Comment:
I wouldn't use one weight element for all entities. Standard approach is to use one element for each item. 

That said, something similar to the should be moved inside foreach below. It will leave in form array structure next to "display" and "remove_button".